symbols/constant_current_diode_vert.ts

```diff
@@ -0,0 +1,20 @@
+import { rotateSymbol } from "drawing/rotateSymbol"
+import constant_current_diode_horz from "./constant_current_diode_horz"
+
+const rotatedSymbol = rotateSymbol(constant_current_diode_horz)
+
+const texts = rotatedSymbol.primitives.filter((p) => p.type === "text")
+
+const val = texts.find((t) => t.text === "{VAL}")
```

Comment:
@aman1376 To fix type check, add the exclamation mark at the end.

`const val = texts.find((t) => t.text === "{VAL}")!`